ledger-tool/src/main.rs

```diff
@@ -778,6 +835,13 @@ fn main() {
                     .help("Include sysvars too"),
             )
             .arg(&max_genesis_archive_unpacked_size_arg)
+            .arg(
+                Arg::with_name("account_data_output")
+                    .long("account-data-output")
```

Comment:
same as above, "format" plus a `.default_value()`?
```suggestion
                Arg::with_name("format")
```